projects/cds/src/merchandising/adapters/strategy/cds-merchandising-strategy.adapter.spec.ts

```diff
@@ -9,11 +9,12 @@ import { StrategyProducts } from '../../model/strategy-products.model';
 import { CdsMerchandisingStrategyAdapter } from './cds-merchandising-strategy.adapter';
 import createSpy = jasmine.createSpy;
 
+const uuidv4 = require('uuid/v4');
```

Comment:
Can we just mock the UUID, or is there a need to generate a proper one?

Reply:
I've changed it to a string literal